packages/core/admin/admin/src/components/NpsSurvey/index.js

```diff
@@ -1,14 +1,52 @@
-import * as React from 'react';
-
-import { Flex, Button, Typography } from '@strapi/design-system';
-import { usePersistentState } from '@strapi/helper-plugin';
+import React, { useState } from 'react';
+
+import { Box, Flex, IconButton, Button, Typography, Textarea, Portal } from '@strapi/design-system';
+import { Cross } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { useNpsSurveySettings } from './hooks/useNpsSurveySettings';
+
+const BannerWrapper = styled(Flex)`
+  border: 1px solid ${({ theme }) => theme.colors.primary200};
+  background: ${({ theme }) => theme.colors.neutral0};
+  box-shadow: ${({ theme }) => theme.shadows.filterShadow};
+  padding: ${({ theme }) => theme.spaces[4]};
+  flex-direction: column;
+  position: fixed;
```

Comment:
I think a lot of these can be applied as props and should be, check out this file for more help – https://github.com/strapi/design-system/blob/8235bff6ec681c5dd70b06786bf11d4943b21e99/packages/strapi-design-system/src/Flex/Flex.tsx#L13

Let me know if you're not familiar with TS and we can chat more about it 💪🏼

Reply:
I thought its bit difficult to read what all styled are applied if they are half inlined(props) and half under styled component. But will go with the props which are exposed explicitly for the purpose.